app/helpers/icons_helper.rb

```diff
@@ -91,6 +91,10 @@ def like_icon
     icon('fas', 'heart')
   end
 
+  def plantpart_icon(name)
```

Comment:
```suggestion
  def plant_part_icon(name)
```
Simply for the consistency, not a biggie...